enterprise/api/v1/serializers.py

```diff
@@ -210,6 +209,11 @@ def to_representation(self, instance):
         # Add the Enterprise enrollment URL to each content item returned from the discovery service.
         for item in search_results:
             content_type = item['content_type']
+            marketing_url = item.get('marketing_url')
+            if marketing_url:
+                item['marketing_url'] = utils.update_query_parameters(
+                    marketing_url, utils.get_enterprise_utm_context(enterprise_customer)
+                )
             if content_type == 'courserun':
```

Comment:
Move the comment above the `for` block down here closer to where we are populating the enrollment URL.